### ISSUE TITLE
LArray.divnot0 does not work when dtype=object

### ISSUE THREAD

**Issue by gdementen** [larray-project/larray#457]

**Comment by alixdamman** [comment]
I can't reproduce the bug. Maybe linked to old version of Numpy?

I made a simple test:
```python
    >>> num = ndtest((3, 3))
    >>> den = num.copy()
    >>> den['a0'] = 0
    >>> num = num.astype('O')
    >>> res = num.divnot0(den)
    >>> res
   a\b   b0   b1   b2
     a0  0.0  0.0  0.0
     a1  1.0  1.0  1.0
     a2  1.0  1.0  1.0
    >>> res.dtype
    object
    >>> den = den.astype('O')
    >>> res = num.divnot0(den)
    >>> res
   a\b   b0   b1   b2
     a0  0.0  0.0  0.0
     a1  1.0  1.0  1.0
     a2  1.0  1.0  1.0
    >>> res.dtype
    object
```
with numpy==1.16.3

**Comment by gdementen** [comment]
This is not related to the version of numpy but of larray 😃 (I can reproduce it using larray 0.29) but not with master, so I **guess** it was fixed in 5f626d4386a722bdd6f43cf16a2401f7e3851c36 as a by-product of fixing #705. We **might** want to add a note in the changelog for 0.30 about this. Unsure it's worth it >> your call.

**Comment by alixdamman** [comment]
I honestly don't think we need to add a changelog for that. 
So I close the issue.
It is a good surprise to note that 2 issues has been fixed in one shot 😃
